lib/core/src/lib/auth/oidc/oidc-authentication.service.ts

```diff
@@ -125,20 +123,25 @@ export class OidcAuthenticationService extends BaseAuthenticationService {
         });
     }
 
+    /**
+     * Gets the username of the authenticated user.
+     *
+     * @returns the logged username
+     */
     getUsername() {
```

Comment:
Could you please update documentation (md) file for this? It's called authentication.service.md file. I see there some changes but not everything was updated, for example that getUsername function is missing there.